Backend/src/main/java/ktsnvt/tim1/DTOs/ReservationTypeDTO.java

```diff
@@ -0,0 +1,5 @@
+package ktsnvt.tim1.DTOs;
+
+public enum ReservationTypeDTO {
+    all,bought,reserved
```

Comment:
Enum members are constants and their text should be all-uppercase

Reply:
Changed to uppercase.